tests/phpunit/TestFramework/Coverage/JUnit/JUnitTestFileDataProviderTest.php

```diff
@@ -110,4 +110,43 @@ public function test_it_works_with_feature_junit_format(): void
 
         $this->assertSame('/codeception/tests/bdd/FeatureA.feature', $testFileInfo->path);
     }
+
+    /**
+     * @dataProvider xmlProvider
+     */
+    public function test_it_does_not_trigger_count_assertion(string $xml): void
```

Comment:
Not sure to understand the test case

Reply:
Broken query triggers an assertion I added in the tested class.